device-types/Fortinet/FG-60D.yml

```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: Fortinet
+model: FortiGate 60D
+slug: fg-60d
```

Comment:
Please append `fortinet-` to the slug

Reply:
most Fortinet devices don't append it, and all Fortigate are without it, so I prefer to keep it this way.

Reply:
Thanks for this clarification. I'll do it.